lib/extends/initiative_model_extend.rb

```diff
@@ -1,24 +0,0 @@
-# frozen_string_literal: true
-require "active_support/concern"
-
-module InitiativeModelExtend
-  extend ActiveSupport::Concern
-
-  included do
```

Comment:
Thanks @armandfardeau ! Just a quick question: why didn't you copy the original lines that were deleted in this file by https://github.com/OpenSourcePolitics/decidim-aws/commit/c9419cc543c1c44e9255e3bfa13bb8c5279aab5e# ? 

<img width="1384" alt="Capture d’écran 2021-05-19 à 15 53 29" src="https://user-images.githubusercontent.com/52420208/118824755-63a33400-b8ba-11eb-9078-12d22ccc7f0b.png">

Reply:
This delegation doesn't seem useful anymore.